bindata/manifests/plugins/sriov-cni.yaml

```diff
@@ -27,6 +27,12 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: sriov-cni
+      initContainers:
```

Comment:
Why using initContainter for ib-cni? Cann't we use the same approach as the sriov-cni?

Reply:
To save system resource, With initContainer, we just use one daemonset for all CNI binary copy, this is the same as how Multus related CNI binaries are copied in cluster network operator.